examples/handshake.rs

```diff
@@ -0,0 +1,102 @@
+use rand::Rng;
+use std::io::Write;
+use std::net::{IpAddr, Ipv4Addr, Shutdown, SocketAddr, TcpStream};
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use bitcoin::consensus::encode;
+use bitcoin::network::address;
+use bitcoin::network::constants;
+use bitcoin::network::message;
+use bitcoin::network::message_network;
+use bitcoin::network::stream_reader::StreamReader;
+
+fn main() {
+    // This example establishes a connection to a Bitcoin node, sends the intial
+    // "version" message, waits for the reply, and finally closes the connection.
+
+    let address = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(130, 149, 80, 221)), 8333);
```

Comment:
What IP is that? 
I'm not sure it's a good idea to put hardcoded actual IPs here, we might cause an accidental DOS on that address

Reply:
That is the address of a public Bitcoin node we run at TU Berlin. I doubt it couldn't handle some connections from people trying this example. 

I put it in there because I wanted to supply a solution that would directly work. However, I could rewrite this so the IP is supplied via an argument. Should I?

Reply:
The example now takes the address of a Bitcoin peer as an argument.